cmd/configrepo/syntax.go

```diff
@@ -67,6 +62,20 @@ func (sr *SyntaxRunner) Run(args []string) {
 	}
 }
 
+func (sr *SyntaxRunner) FindPluginJar() {
+	if found, err := plugins.PluginById(PluginId, PluginDir); err != nil {
+		utils.Echofln("%s", err)
```

Comment:
Generally, we should output to STDERR using `Errfln()` instead. I may make some changes here to increase helpful output/messages to the user, particularly in the case of a custom `--plugin-id`.